keyboards/winkeyless/bface/bface.h

```diff
@@ -34,7 +34,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
   { K02,   K12,   K22,   K32,   K42,   K52,   KC_NO, KC_NO, KC_NO, KC_NO, KA2,   KB2,   KC_NO, KD2,   KC_NO }, \
   { K03,   K13,   K23,   K33,   K43,   K53,   KC_NO, KC_NO, KC_NO, KC_NO, KA3,   KB3,   KC3,   KD3,   KC_NO }, \
   { K04,   K14,   K24,   K34,   K44,   K54,   KC_NO, KC_NO, KC_NO, KC_NO, KA4,   KB4,   KC4,   KC_NO, KE4   }, \
-  { KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO }, \
+  { K04,   KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO }, \
```

Comment:
This isn't right.  The line above this starts with K04, which means that this will be the same key.  I'm not sure that's the intent

Reply:
@drashna I don't think this change would affect the currently supported PCB revision(s). The original matrix submitted in #2493 hasn't changed at all since then (opened March 7, 2018 and merged August 19, 2018). It seems the common revision of this PCB works fine on the existing codebase, but this user has a less-common revision with a slightly different matrix. The row-column intersection for the Escape key on one revision wouldn't exist on the other, so there shouldn't be any breakage between the two, but it would be great if we could get someone running on the existing codebase could test this.

Reply:
My intent was supporting two revision of B.face board with one code. I did because I did not know the way to handle this kind of problem. If there is any recommended way to fix this problem, I'll recreate my patch using that way. @drashna

Reply:
This should actually be K05 not K04. 

Also we typically do keyboard_name/rev_number to differentiate between the different pcbs.

Reply:
Changing the identifier on Line 37 from `K04` to `K05` prevents the LAYOUT_60_ansi macro from compiling because the argument list doesn't contain a `K05`:

```shell
Compiling: keyboards/winkeyless/bface/keymaps/default/keymap.c                                     In file included from keyboards/winkeyless/bface/keymaps/default/keymap.c:18:0:
keyboards/winkeyless/bface/bface.h:37:5: error: 'K05' undeclared here (not in a function)
   { K05,   KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO, KC_NO }, \
     ^
keyboards/winkeyless/bface/keymaps/default/keymap.c:21:11: note: in expansion of macro 'LAYOUT_60_ansi'
     [0] = LAYOUT_60_ansi(
           ^
 [ERRORS]
 |
 |
 |
make[1]: *** [tmk_core/rules.mk:377: .build/obj_winkeyless_bface_default/keyboards/winkeyless/bface/keymaps/default/keymap.o] Error 1
make[1]: *** Waiting for unfinished jobs....
Compiling: quantum/keymap_common.c                                                                  [OK]
Compiling: quantum/quantum.c                                                                        [OK]
Make finished with errors
make: *** [Makefile:553: winkeyless/bface:default] Error 1
```

Forking the codebase into bface/rev0p1 and bface/rev0p2 or whatever is appropriate – do we know the version number of the common PCB revision? – is one solution, but IMO the current implementation is much cleaner, and would be fine if we can confirm it doesn't break anything with currently-supported PCB revisions.

Reply:
If anyone knows other b.face revs, I can move current base into appropriate subdir.